themes/mlibrary_new/common/header.php

```diff
@@ -58,8 +58,8 @@
         </div>
     </div>
 
-    <?php if ((get_theme_option('display_header') !== '0')): ?>
-    <header id="banner" class="<?php echo get_theme_option('header_flow'); ?> page-header" style="background-size:cover;background-image:url('<?php 
+<?php if (((get_theme_option('display_header') !== '0')) and (end(explode('/',$_SERVER['REQUEST_URI']))!='about')) : ?>   
```

Comment:
I'm uncomfortable with inspecting the REQUEST_URI in the theme.  It looks like there's a filter on `theme_options` you can use to change the option on `display_header` for only some pages.  That would be better.

Reply:
The about page generated Auto from a simple page plugin. So I do not have an access to the page to use the display_header. I used the display_header for other pages I have an access to the php file. Any ideas how to handle this for auto-generated pages?